python/equistore-core/equistore/core/labels.py

```diff
@@ -492,6 +492,40 @@ def insert(self, index: int, name: str, values: np.ndarray) -> "Labels":
 
         return Labels(names=new_names, values=new_values)
 
+    def permute(self, dims: List[int]) -> "Labels":
```

Comment:
naming suggestions
```suggestion
    def permute(self, dimensions: List[int]) -> "Labels":
```

```suggestion
    def permute(self, dimensions_indexes: List[int]) -> "Labels":
```

```suggestion
    def permute(self, columns: List[int]) -> "Labels":
```

Reply:
Lets go with the second one. Even though it is the longest. For me `dimensions` are more the actual Labels or the `names`.